docs/user-flows.md

```diff
@@ -1,262 +1,153 @@
+  const flow = await lighthouse.startFlow(page);
 
   await flow.beginTimespan();
-  await page.type('#username', 'lighthouse');
   await page.type('#password', 'L1ghth0useR0cks!');
   await page.click('#login');
   await page.waitForSelector('#dashboard');
   await flow.endTimespan();
 
   await browser.close();
-
   writeFileSync('report.html', await flow.generateReport());
-}
-
-main();
+})();
 ```
-</details>
-<br>
 
 ### Snapshot
 
-Snapshot reports analyze the page in a particular state, typically after the user has interacted with it, and have similar use cases to the Elements Panel in DevTools.
-
-#### Benefits
-
-- Analyzes the page in its current state.
-
-#### Limitations
-
-- Does not provide an overall performance score or metrics.
-- Cannot analyze any issues outside the current DOM (e.g. no network, main-thread, or performance analysis).
-
-#### Use Cases
-
-- Find accessibility issues in single page applications or complex forms.
-- Evaluate best practices of menus and UI elements hidden behind interaction.
-
-#### How to use
+In DevTools, select "Snapshot" as the mode. Set up the page in the state you want to evaluate. Then, click _Analyze page state_.
 
-<details>
-<summary>
-DevTools
-</summary>
+![snapshotdt](https://user-images.githubusercontent.com/39191/168929172-92a70108-a053-4dda-b719-2900b9d3d956.png)
 
-1. Go to the page you want to test
-2. Interact with the page so it's in a state you want to test
-3. Select "Snapshot" as your mode
-4. Click "Analyze page state".
+#### Snapshots in the Node.js API
 
-> Note: DevTools only generates a report for a standalone snapshot, it cannot be combined with other steps in a user flow report.
-
-<img width="1203" alt="Screen Shot 2022-05-16 at 1 30 08 PM" src="https://user-images.githubusercontent.com/6752989/168677313-8be0591a-8e17-488c-b602-b47e487a75a3.png">
-</details>
-
-<details>
-<summary>Node API</summary>
 
 ```js
 import {writeFileSync} from 'fs';
 import puppeteer from 'puppeteer';
-import api from 'lighthouse/lighthouse-core/fraggle-rock/api.js';
+import lighthouse from 'lighthouse/lighthouse-core/fraggle-rock/api.js';
 
-async function main() {
+(async function() {
   const browser = await puppeteer.launch();
   const page = await browser.newPage();
   await page.goto('https://example.com');
-  const flow = await api.startFlow(page);
+  const flow = await lighthouse.startFlow(page);
 
   await page.click('#expand-sidebar');
   await flow.snapshot();
 
   await browser.close();
-
   writeFileSync('report.html', await flow.generateReport());
-}
-
-main();
+})();
 ```
-</details>
-<br>
 
 ## Creating a Flow
 
-So far we've seen individual Lighthouse modes in action. The true power of flows comes from combining these building blocks into a comprehensive flow to capture the user's entire experience.
-
-### Selecting Boundaries
+<img src="https://user-images.githubusercontent.com/39191/168931109-5ae80620-7464-4d98-a834-71169a5b6a61.png" height=300>
```

Comment:
Can this guy be a little smaller, maybe 200px?

Reply:
matches the rest now. 240px